sites/main-site/src/content/blog/2024/seqera-containers-part-1.mdx

```diff
@@ -0,0 +1,453 @@
+
+import butwhy from '@assets/images/blog/seqera-containers-part-1/simone-secci-49uySSA678U-unsplash.jpg';
+import mulled from '@assets/images/blog/seqera-containers-part-1/hannah-pemberton-bXi4eg4jyuU-unsplash.jpg';
+import chillies from '@assets/images/blog/seqera-containers-part-1/anne-nygard-3tPkwnNRRkI-unsplash.jpg';
+import wave_logs from '@assets/images/blog/seqera-containers-part-1/logs_screenshot.png';
+import leaves from '@assets/images/blog/seqera-containers-part-1/providence-doucet-mE5MBZX5sko-unsplash.jpg';
+
+import { Image } from 'astro:assets';
+
+## Introduction
+
+Dear nf-core community, the core team would like to inform you about some upcoming changes in how we would like to handle software containers... :package:
+
+Software containers are a fundamental part of modern bioinformatics workflows.
+Nextflow supports multiple container platforms, but the two most commonly used are Docker and Singularity.
+When using containers, all software requirements for a specific Nextflow process are wrapped up into an image and referenced within the pipeline code.
+The process tasks run in isolation in the host environment, and the end user doesn't need to worry about installing software dependencies for the pipeline.
+The software builds are locked in time, making results highly reproducible over many years.
+
+To ensure maximum compatibility across different environments, nf-core modules are typically set up with a conda `environment.yml`, listing software package dependencies from [Bioconda](https://bioconda.github.io/).
+The [BioContainers project](https://biocontainers.pro/) conveniently builds Docker and Singularity images for all tools on Bioconda automatically, hosting the images publicly on [quay.io](quay.io) and the Galaxy FTP servers, respectively.
+This means that the nf-core module developer can find the matching [BioContainer](https://biocontainers.pro/) images for their Bioconda package, and then add the image URLs into the module's `main.nf` script.
+
+The Bioconda and BioContainer projects have been invaluable to nf-core's success.
+We're hugely grateful to all contributors for their work and to the Galaxy project for hosting.
+
+## Change is on the breeze
+
+<Image class="d-block ms-3 mb-3 float-end rounded w-25 shadow" src={butwhy} alt="Photo by Simone Secci on Unsplash." />
+
+Bioconda and BioContainers have worked very well for nf-core.
+However, we are now looking to migrate to a new system: Seqera Containers.
+
+The motivation comes down to a few key reasons:
+
+-   Difficulties with BioContainers
+    -   [Mulled (multi-package) images](#mulled-multi-package-images)
+    -   [Slow Singularity image availability](#time-to-singularity-image)
+    -   [BioContainers API](#biocontainers-api)
+    -   [Reliability of hosting](#reliability-of-hosting)
+-   New features we'd like
+    -   [Conda lock files](#exceptionally-reproducible)
+    -   Simplified developer workflow
```

Comment:
It is not weird that this point does not have its own paragraph in the post like all the others?

Reply:
Yeah, but the only one I could think of was "What this means to you > Developers" and that's right at the end, so not that useful. I don't think it's _that_ bad.